boards/tsi/tsi/tsi_mimx8ml8_adsp.dts

```diff
@@ -14,11 +14,17 @@
 	compatible = "nxp";
 
 	chosen {
-		zephyr,sram = &sram0;
+		//zephyr,sram = &sram0;
+		zephyr,sram = &sram_txe;
 
 		zephyr,console = &uart4;
 		zephyr,shell-uart = &uart4;
 	};
+	sram_txe: memory@60040000 {
+		device_type = "memory";
+		compatible = "mmio-sram";
+		reg = <0x60040000 DT_SIZE_K(512)>;
```

Comment:
We can call this as tsi_txe.dts, instead of calling it tsi_mimx8ml8_adsp ? is that doable ? I can give you pointers on how to add it. Also this current sram_txe we were thinking will be sram1 so we can call it memory@60640000 and change reg to 0x60640000